src/sql/workbench/api/common/extHostDataProtocol.ts

```diff
@@ -27,7 +27,7 @@ export class ExtHostDataProtocol extends ExtHostDataProtocolShape {
 
 	constructor(
 		mainContext: IMainContext,
-		private uriTransformer: IURITransformer
+		private uriTransformer: IURITransformer | null
```

Comment:
I think we're trying to use `undefined` instead of `null` in this code base.  Is there a reason we need to use null here?